File: CodeWars/8kyu/CPP/CountOfPositivesSumOfNegatives.cpp
// src: https://www.codewars.com/kata/576bb71bbbcf0951d5000044/
#include <vector>

std::vector<int> countPositivesSumNegatives(std::vector<int> input)
{
  if (input.size() == 0) return {};
    int cant_positive = 0, sum_negative = 0;
    for(auto i: input){
      if ( i > 0) cant_positive++;
      if ( i < 0) sum_negative += i;
    }
  return {cant_positive, sum_negative};
}

File: CodeWars/8kyu/Python/CountOfPositivesSumOfNegatives.py
# src: https://www.codewars.com/kata/576bb71bbbcf0951d5000044/
def count_positives_sum_negatives(arr):
    if len(arr) == 0:
        return []
    cant_positive = 0
    sum = 0
    for i in arr:
        if i < 0:
            sum += i
        elif i > 0:
            cant_positive += 1
    return [cant_positive,sum]